sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -595,7 +595,15 @@ case class WeekOfYear(child: Expression) extends UnaryExpression with ImplicitCa
 
 // scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(timestamp, fmt) - Converts `timestamp` to a value of string in the format specified by the date format `fmt`.",
+  usage = """
+    _FUNC_(timestamp, fmt) - Converts `timestamp` to a value of string in the format specified by the date format `fmt`.
+    See `java.time.format.DateTimeFormatter` for valid date and time format patterns.
+  """,
+  arguments = """
+    Arguments:
+      * timestamp - Date/timestamp or String to be converted to the given format. Default value is current time
+      * fmt - Date/time format pattern to follow
```

Comment:
Could you move `See `java.time.format.DateTimeFormatter` for valid date and time format patterns.` here.